### DISCUSSION TITLE
How do I add page transitions with Inertia and vue?

### DISCUSSION THREAD

**Discussion by ApolloDevs** [inertiajs/inertia#810] [Help (Vue)]
I have attempted a few examples that I have found online but sadly I haven't really got anywhere. I'm using the Laravel adapter with InertiaJS and Vue and TailwindCSS. 

Say I have the following layout - 
```
<AppLayout>
<ProfileSidebar />
<Slot></Slot>
<AppLayout />
```

How would I make the items inside the slot transition when selecting a new page from the sidebar? I've tried the <transition> class with CSS in a <style> tag in the component but no transitions take place. (with <transition> wrapped around the slot!) Looking for a basic example to get me off the ground!

Thank you.

**Comment by ajnsn** [comment]
@ApolloDevs Try to wrap your slot with a Vue transition, e.g. 

```js
<transition name="fade" mode="out-in" appear>
    <slot />
</transition>
```

CSS for Vue 3

```css
.fade-enter-active,
.fade-leave-active {
    @apply transition-opacity;
}

.fade-enter-from,
.fade-leave-to {
    @apply opacity-0;
}
```
